### PR TITLE
Amp 58497 go ampli developer doc

### PR DESCRIPTION
**Pull request by justin-fiedler**
# Amplitude Developer Docs PR

## Description
* Go doc updates for latest changes
* Go Ampli docs

## Change type
- [x] Doc update.
- [x] New documentation.

@amplitude-dev-docs
@caseyamp

### CODE REVIEW

File: docs/data/sdks/go/ampli.md [hunk 1]
```diff
@@ -1,18 +1,14 @@
 ---
-title: Go Ampli Wrapper
+title: Go Ampli Wrapper (Alpha)
```
Comment (yuhao900914) [inline docs/data/sdks/go/ampli.md:2]: Change this to Beta?

File: docs/data/sdks/go/ampli.md [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
-    [:material-github: Github](https://github.com/amplitude/analytics-go) · [:material-code-tags-check: Releases](https://github.com/amplitude/analytics-go/releases) · [:material-book: API Reference](https://pkg.go.dev/github.com/amplitude/analytics-go/amplitude)
-
 Amplitude Data supports tracking analytics events from Go apps. The generated tracking library is packaged as a Go package.
 
 The tracking library exposes a type-safe function for every event in your team’s tracking plan. The function’s arguments correspond to the event’s properties and are strongly typed to allow for auto code completion.
 
-!!!tip
-
-    See example apps that use the Go runtimes on [GitHub](https://github.com/amplitude/ampli-examples/tree/main/Go).
+!!!alpha "Go Ampli Resources (Alpha)"
```
Comment (yuhao900914) [inline docs/data/sdks/go/ampli.md:10]: Remove the alpha tag?

File: docs/data/sdks/go/index.md [hunk 3]
```diff
@@ -4,12 +4,13 @@ description: The Amplitude Go SDK installation and quick start guide.
 icon: simple/go
 ---
 
-!!!alpha "Alpha SDK Resources"
-    [:material-github: Github](https://github.com/amplitude/analytics-go) · [:material-code-tags-check: Releases](https://github.com/amplitude/analytics-go/releases) · [:material-book: API Reference](https://pkg.go.dev/github.com/amplitude/analytics-go/amplitude)
+The Go SDK lets you send events to Amplitude. This library is open-source, check it out on [GitHub](https://github.com/amplitude/analytics-go).
 
---8<-- "includes/no-ampli.md"
+!!!alpha "Go SDK Resources (Alpha)"
```
Comment (yuhao900914) [inline docs/data/sdks/go/index.md:9]: Alpha tag

File: docs/data/sdks/go/ampli.md [hunk 4]
```diff
@@ -62,7 +58,7 @@ Initialize Ampli in your code. The `Load()` method accepts configuration option
 
 ```Go
 import (
-	"ampli-example/ampli"
+    "<your-module-name>/ampli"
```
Comment (yuhao900914) [inline docs/data/sdks/go/ampli.md:61]: extra line above?

**Review by yuhao900914 — CHANGES_REQUESTED** [review]

File: docs/data/sdks/go/index.md [hunk 5]
```diff
@@ -70,18 +71,21 @@ Events represent how users interact with your application. For example, "Button
 // Track a basic event
 // EventOne of UserID and DeviceID is required as well as EventType
 client.Track(amplitude.Event{
-  EventType:    "Button Clicked",
-  EventOptions: amplitude.EventOptions{UserID: "user-id"},
+    UserID: "user-id",
+    EventType:    "Button Clicked",
```
Comment (liuyang1520) [inline docs/data/sdks/go/index.md:75]: Minor, alignment?
```
UserID:    "user-id",
EventType: "Button Clicked",
```

**Review by liuyang1520 — APPROVED** [review]

**Review by yuhao900914 — APPROVED** [review]

**Pull request merged**